docs/expose.md

```diff
@@ -4,19 +4,19 @@ The Operator provides entry points for accessing the database by client applicat
 
 This document describes the usage of [Custom Resource manifest options](operator.md) to expose clusters deployed with the Operator. 
 
-## Using single entry point in a sharded cluster
+## Using a single entry point in a sharded cluster
 
-If Percona Server for MongoDB [sharding mode](sharding.md) is turned **on** (default behavior), then database cluster runs special
-`mongos` Pods - query routers, which acts as an entry point for client applications:
+If Percona Server for MongoDB [sharding mode](sharding.md) is turned **on** (the default behavior), then database cluster runs special
```

Comment:
I think "the default behavior" should just remain as "default behavior" without the "the" - that's what my opensource English feelings tell me.